INSTALL.md

```diff
@@ -0,0 +1,162 @@
+# Installationsguide för Baga Aqua System API
+
+### 2019-05-21
+
+# Förord
+
+Denna Installationsguide är för Baga Aqua Systems API. Systemet är byggt i NodeJS med MongoDB som databas. Arbetet är utfört av studenter på Blekinge Tekniska Högskola som en del i kursen PA1416 under vårenterminen 2019 åt Baga Water Technology AB. Denna guide är till för att underlätta vid installation av systemet. Läsaren förväntas ha kunskaper om den egna miljön som installationen sker i.
```

Comment:
As the rest of the installation guide uses language that is directed directly towards the reader, it is recommended to change "läsaren" to "du"

Reply:
fixed in commit 7127c28